src/server/api/controllers/users.controller.js

```diff
@@ -39,14 +65,9 @@ const editUser = async (UserId, updatedUser) => {
   });
 };
 
-
-const saveUser = async (data) => {
-  await knex('users').insert(data);
-};
-
```

Comment:
I think this PR is trying to delete user post. You have to remove this changes from PR. If you merge it then user post will be delete from develop branch.

Reply:
Ok i think its due to merge conflict. I will update this. Thanks @semih1239  👍